docs/enhancements/api-for-skipping-validations.md

```diff
@@ -0,0 +1,153 @@
+---
+title: api-for-skipping-validations
+authors:
+  - "@pmaidmen"
+creation-date: 2023-01-08
+last-updated: 2023-01-08
+---
+
+# Allow validations to be skipped
+
+# Summary
+
+We need a new api that will allow us to skip cluster/host validations, this will be for unsupported use.
+This API should work on a per cluster basis and will have it's own feature flag to allow us to enable it for specific users.
+An event should be emitted whenever the API is used during an installation, this can be used to drive signatures etc.
```

Comment:
I meant that we can write a signature to look for these events and that this could be used on a Triage ticket to indicate that the user had overridden validations in this particular case.

I may remove this line completely from the document as there is feedback later that suggests that we should not even create triage tickets for clusters that have emitted such events.